source/includes/administration/_reports_pricing_v2.md

```diff
@@ -0,0 +1,178 @@
+                "name": {
+                  "en": "Public IP",
+                  "fr": "Public IP"
+                },
+                "cost": "432.00",
+                "usage": "432.0",
+                "price": "1.0",
+                "period": "HOUR",
+                "unit": {
+                  "unit": "HOUR",
+                  "name": {}
+                }
+              }
+            ],
+            "subTotal": "432.00"
+          }
+        ]
+      }
+    ],
+    "startDate": "2021-03-30T00:00:00Z",
+    "endDate": "2021-04-01T23:59:59.999Z",
+    "reportGenerated": true
+  }
+}
+
+```
+
+Organization Report Attributes | &nbsp;
+---------- | -----------
+`currencies`<br/>*Array[Object]* | Returns an array of currencies for the usage.
+`currencies.currency`<br/>*string* | The short-name of the currency.
+`currencies.total`<br/>*string* | A string containing the total of all usage.
+`currencies.categories`<br/>*Array[Object]* | A array of category objects. Contains all categories that had usage for the period. 
+`currencies.categories.name`<br/>*Object* | A map of short language codes to their translated category names. 
+`currencies.categories.subTotal`<br/>*string* | A string with the subtotal of usage for the category. It is the summed up product usage.
+`currencies.categories.products`<br/>*Object* | A map of short language codes to their translated category names. 
+`currencies.categories.products.sku`<br/>*string* | A unique string representing the sku for a product.
+`currencies.categories.products.name`<br/>*Object* | The name object in each language for the product name.
+`currencies.categories.products.cost`<br/>*string* | A string of the summed up total cost for the product.
+`currencies.categories.products.usage`<br/>*Object* | A string of the summed up total usage for the product.
+`currencies.categories.products.price`<br/>*Object* | A string representing the *average* unit price over the preriod.
+`currencies.categories.products.period`<br/>*string* | The period for the product capture. Possible values: HOURS, MONTH.
+`currencies.categories.products.unit`<br/>*Object* | The unit object of the product.
+`currencies.categories.products.unit.unit`<br/>*Object* | The unit value of the product.
+`currencies.categories.products.unit.name`<br/>*Object* | The name of the unit of the product in the required language. Only present when defining custom units.
+`startDate`<br/>*string* | An ISO-8601 instant format string representing the start of the report.
+`endDate`<br/>*string* | An ISO-8601 instant format string representing the end of the report.
+`reportGenerated`<br/>*boolean* | Whether or not a report could be generated for this time period.
+
+<!------------------- GET CUSTOMER REPORT --------------------->
+
+### Get a customer's report
+
+`GET /reports/customers?start_date:=end_date=&end_date=:end_date`
+
+Retrieves a list of usage all usages for the provided reseller organization and all of it's suborgs.
```

Comment:
Retrieves a list of all usages

Reply:
6c2e3a9f91f7e73660ad634a3263df65853ce5ee